RecoEgamma/EgammaElectronAlgos/src/FTSFromVertexToPointFactory.cc

```diff
@@ -48,7 +49,7 @@ FreeTrajectoryState FTSFromVertexToPointFactory::get( MagneticField const & magF
   auto pyNew =  -sa*pxOld + ca*pyOld;
   GlobalVector pNew(pxNew, pyNew, pz);  
 
-  GlobalTrajectoryParameters gp(xmeas, pNew, charge, & magField);
+  GlobalTrajectoryParameters gp(xmeas, pNew, charge, & magField, std::move(magFieldAtPoint));
```

Comment:
is move really needed at this point?
Did you see a time improvement here?

Reply:
no its not needed, I can back it out, I just figured lets send it a && and if the code can ever deal with it in the future it'll pick it up. Its not costly? Do not have a strong opinon

Reply:
fine to leave it here.
IIUC, the GlobalVector does not have a move constructor. So the move here likely does nothing and prevents potential future use of magFieldAtPoint below this line.
OTOH, everything is inlined and the story of copying likely doesn't matter.